modules/Microsoft.Compute/galleries/images/deploy.bicep

```
@@ -54,12 +54,25 @@ param minRecommendedMemory int = 4
 @maxValue(4000)
 param maxRecommendedMemory int = 16
 
-@description('Optional. The hypervisor generation of the Virtual Machine. Applicable to OS disks only. - V1 or V2.')
+@description('''Optional. The hypervisor generation of the Virtual Machine. Can be either V1 or V2.
```

```suggestion
@description('''Optional. The hypervisor generation of the Virtual Machine.
```

Was is allowed is already covered by the allowed set